hqscripts/management/commands/makemigrations.py

```diff
@@ -71,7 +71,7 @@ def assert_exclusive_options(opt_name, extra=[]):
 
     def write_migration_files(self, changes):
         super().write_migration_files(changes)
-        self.write_migrations_lock()
+        self.write_migrations_lock(self.dry_run)
```

Comment:
nit: could simplify this and not affect that method signature
```suggestion
        if self.dry_run:
            self.write_migrations_lock()
```

Reply:
That doesn't work because `super().handle()` is what sets `self.dry_run`, but this handle method has a mode where it doesn't call that ([here](https://github.com/dimagi/commcare-hq/blob/1646d6df9d2a0a74acd3451d9d81e651105f85f4/hqscripts/management/commands/makemigrations.py#L68)).